src/Meditation.PatchingService/Services/PatchApplier.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Meditation.InjectorService;
+using Meditation.PatchingService.Models;
+using System;
+
+namespace Meditation.PatchingService.Services
+{
+    internal class PatchApplier : IPatchApplier
+    {
+        private readonly IProcessInjector _processInjector;
+        private readonly IProcessInjecteeExecutor _processInjecteeExecutor;
+
+        public PatchApplier(IProcessInjector processInjector, IProcessInjecteeExecutor processInjecteeExecutor)
+        {
+            _processInjector = processInjector;
+            _processInjecteeExecutor = processInjecteeExecutor;
+        }
+
+        public void ApplyPatch(int pid, PatchInfo patch)
+        {
+            const string nativeHookEntryPoint = "MeditationInitialize";
+            const string nativeHookLibrary = @"Meditation.Bootstrap.Native.dll";
```

Comment:
Mb better to declare constants not in method, but create separate nativeHookSpec that will provide it